test_runner/fixtures/neon_fixtures.py

```diff
@@ -4600,7 +4600,6 @@ def scrubber_cli(
         ]
         args = base_args + args
 
-        log.info(f"Invoking scrubber command {args} with env: {env}")
```

Comment:
Maybe just removing the env logging, but leaving the args the scrubber was run with?

Reply:
Agree, but this log line does not look so important. It seems nobody looked at it from the beginning.

Reply:
updated the line